scripts/build/termux_step_start_build.sh

```diff
@@ -47,6 +49,28 @@ termux_step_start_build() {
 		fi
 	fi
 
+	if [ "$TERMUX_PKG_BUILD_IN_SRC" = "true" ]; then
+		echo "Building in src due to TERMUX_PKG_BUILD_IN_SRC being set to true" > "$TERMUX_PKG_BUILDDIR/BUILDING_IN_SRC.txt"
+		TERMUX_PKG_BUILDDIR=$TERMUX_PKG_SRCDIR
+	fi
```

Comment:
No need to move the first line creating that file up, it would have been set the first time.